src/mmw/js/src/draw/templates/splash.html

```diff
@@ -0,0 +1,34 @@
+<h1>
+    Model stormwater and its impact on an area
+</h1>
+
+<p>
+Analyze real geo-data, model storms, and compare conservation or development scenarios in a watershed.
```

Comment:
The markup is all placeholder, going to wait until it is real before sprucing up.